tests/qos/files/qos_params.j2c.yaml

```diff
@@ -504,7 +504,7 @@ qos_params:
                     ecn: 1
                     queue: 3
                     pkts_num_fill_min: 0
-                    pkts_num_trig_ingr_drp: 750848
+                    pkts_num_trig_ingr_drp: 807021
```

Comment:
why is this different from above ?

Reply:
@vmittal-msft , Since the port speed is greater, hence the ingress drop trig pkts are more.